src/WebJobs.Script.WebHost/Properties/Resources.resx

```diff
@@ -183,7 +183,7 @@
   </data>
   <assembly alias="System.Windows.Forms" name="System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
   <data name="Homepage" type="System.Resources.ResXFileRef, System.Windows.Forms">
-    <value>..\Home.html;System.String, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089;utf-8</value>
+    <value>../Home.html;System.String, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089;utf-8</value>
```

Comment:
There's a test that validates the result when hitting the home controller (which uses that resource) and that seems to be passing, so looks like it is happy with the forward slash on Windows.